CHANGELOG.md

```diff
@@ -2,10 +2,44 @@
 
 ## Unreleased
 
+## v0.10.0
+
+# 🎉 OpenTelemetry Collector Contrib v0.10.0 (Beta) 🎉
+
+The OpenTelemetry Collector Contrib contains everything in the [opentelemetry-collector release](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.10.0) (be sure to check the release notes here as well!). Check out the [Getting Started Guide](https://opentelemetry.io/docs/collector/about/) for deployment and configuration information.
+
 ## 🚀 New components 🚀
 - add initial docker stats receiver, without sourcing in top level components (#495)
 - add initial jmx metrics extension structure, without sourcing in top level components (#740)
 - `routing` processor for routing spans based on HTTP headers (#907)
+- `splunkhec` receiver to receive Splunk HEC metrics, traces and logs (#840)
+- Add skeleton for `http_forwarder` extension that forwards HTTP requests to a specified target (#979)
+
+## 💡 Enhancements 💡
+- `stackdriver` exporter
+  - Add timeout parameter (#835)
+  - Add option to configurably set UserAgent string (#758)
+- `signalfx` exporter
+  - Reduce memory allocations for big batches processing (#871)
+  - Add AWSUniqueId and gcp_id generation (#829)
+  - Add delta translator (#839)
+  - Add float support to calculate new metric (#974)
+  - Add cpu delta translations (#954)
+- `metricstransform` processor: Replace `{{version}}` in label values (#876)
+- `resourcedetection` processor: Logs Support (#970)
+- `statsd` receiver: Add parsing for labels and gauges (#903)
+
+## 🧰 Bug fixes 🧰
+- `k8s` processor
+  - Wrap metrics before sending further down the pipeline (#837)
+  - Fix setting attributes on metrics passed from agent (#836)
+- `awsxray` exporter: Fix "pointer to empty string" is not omitted bug (#830)
+- `azuremonitor` exporter: Treat UNSPECIFIED span kind as INTERNAL (#844)
+- `signalfx` exporter
+  - Remove misleading warnings (#869)
+  - Fix producing cloud unique ID dimension (#966)
```

Comment:
This bugfix can be omitted since it's followup for #829 that is mentioned in Enhancements